moviesite/polls/movie_api.py

```diff
@@ -0,0 +1,60 @@
+from mechanize import Browser
+from BeautifulSoup import BeautifulSoup
+import sys
+import re
+import json
+
+def get_data(movie):
+
+    try: 
+        br = Browser()
+        br.open("http://www.imdb.com/find?s=tt&q="+movie)
+        link = list(br.links(url_regex = re.compile(r"/title/tt*")))[0]
+    
+    
+    except:
+        print "Not Found!"
+        exit(3);
```

Comment:
This is not how you should handle errors in a web server. This is what exit does - https://docs.python.org/2/library/constants.html#exit